README.md

```diff
@@ -80,5 +80,5 @@ $ source .venv/Scripts/activate
 ## Example command
 
 ```bash
-$ osivalidator --data data/20210818T150542Z_sv_312_50_one_moving_object.osi --rules rules/
+$ osivalidator --data data/20240221T141700Z_sv_312_10_one_moving_object.osi --rules rules/
```

Comment:
Was the osi version of the trace too old?

Reply:
Add OSI version to file name

Reply:
I corrected the OSI version and added the protobuf version"